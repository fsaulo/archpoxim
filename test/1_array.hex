0xDC000007
0x00000000
0x00000000
0x00000000
0x00000000
0x00000000
0x00000000
0x00000000
0xFC000000
0x000000AA
0x000000BB
0x000000FF
0x00001111
0x0000FFFE
0x00003333
0x00004444
0xABCDEF01
0x12345678
0xF0F0F0F0
0x00000000
0x00000000
0x00000000
0xFFFFFFFF
0xFFFFFFFF
0xFFFF0000
0x0ABC0ABC
0x0ABC0ABC
0x0ABC0ABC
0x0ABC0ABC
0x0ABC0ABC
0xABCDEFAB
0xCDEFABCD
0xEFABCDEF
0xABCDEFAB
0xCDEFABCD
0xEFABCDEF
0xABCDEFAB
0xCDEF0000
0x00000000
0x00000000
0x00000000
