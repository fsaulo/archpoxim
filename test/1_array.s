// Segmento de codigo
.text
    // Tabela de vetor de interrupcao
    init:
        bun main
        .align 5
    // Funcao principal
    main:
        // Finalizacao de execucao
        int 0
// Segmento de dados
.data
    // Exemplos de vetores (alinhamento de 4 bytes)
    V0:
        .byte 0xAA, 0xBB, -1
    V1:
        .2byte 0x1111, -2, 0x3333, 0x4444
    V2:
        .4byte 0xABCDEF01, 0x12345678, 0b11110000111100001111000011110000
    V3:
        .fill 10
    V4:
        .fill 10, 1, -1
    V5:
        .fill 10, 2, 0xABC
    V6:
        .fill 10, 3, 0xABCDEF
    V7:
        .zero 9
