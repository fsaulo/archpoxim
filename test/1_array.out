[START OF SIMULATION]
0x00000000:	bun 7                    	PC=0x00000020
0x00000020:	int 0                    	CR=0x00000000,PC=0x00000000
[END OF SIMULATION]
